castle/cms/browser/controlpanel/status.py

```diff
@@ -0,0 +1,49 @@
+from Products.Five.browser import BrowserView
+import subprocess
+import redis
+import celery
+import celery.bin.base
+import celery.bin.celery
+import celery.platforms
+
+
+class StatusView(BrowserView):
+
+    def docsplit(self):
+        try:
+            subprocess.check_call('docsplit')
+            x = True, 'ok'
+        except OSError as e:
+            x = False, str(e)
+        if x[0]:
```

Comment:
I think can you replace lines 18-21 with just `return x`, because whether `x[0]` is true or false, that's what you're doing...

Reply:
(same below)